packages/client/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@logion/client",
-  "version": "0.32.0-8",
+  "version": "0.32.0-9",
```

For some tests yesterday this version was already published.
```suggestion
  "version": "0.32.0-10",
```